order.rb

```diff
@@ -1 +1,16 @@
 # See README.md
+class Order
+  @@prices = []
+  attr_accessor :customer, :item, :price; :orders; :items; :prices
```

Comment:
I made the @items because I was / am thinking about an added functionality to list the items of one customer in the top customers section, e.g. Nr. 1: Customer 6: Susan Mckenna (Lagos): €553 in  5 orders for the following items: …

Reply:
I don‘t have an exact plan in my head about how exactly I would do it, but I think it still makes sense to use the `@order` variable, because you will need to access the price anyway. By using the orders, you have the advantage of being able to access price and items together and you are sure that they truly belong to each other.

Reply:
yeah you were right, ended up deleting it as it wasn't necessary :)